### DISCUSSION TITLE
Add an area calculation for the fill

### DISCUSSION THREAD

**Discussion by HighW0rks** [Thisal-D/tkchart#15] [Ideas]
If possible I wish to have a function that calculates the area on each grid and adding it for a total until it reaches a specific value and it will give a total area of a chart.

**Comment by thisal-d** [comment]
> If possible I wish to have a function that calculates the area on each grid and adding it for a total until it reaches a specific value and it will give a total area of a chart. 

I added this feature, try it, thanks for the idea you gave.
